src/main/scala/zio/interop/reactivestreams/Adapters.scala

```diff
@@ -289,37 +293,36 @@ object Adapters {
       state.getAndSet(canceled).toNotify.foreach { case (_, p) => p.unsafeDone(IO.fail(())) }
   }
 
-  private def fromPull[R, E, A](zio: ZManaged[R, Nothing, ZIO[R, Option[E], Chunk[A]]])(implicit
+  private def fromPull[R, E, A](zio: ZIO[R with Scope, Nothing, ZIO[R, Option[E], Chunk[A]]])(implicit
     trace: ZTraceElement
   ): ZStream[R, E, A] =
-    unwrapManaged(zio.map(pull => ZStream.repeatZIOChunkOption(pull)))
+    unwrapScoped[R, E, A](zio.map(pull => ZStream.repeatZIOChunkOption(pull)))
 
-  private def unwrapManaged[R, E, A](fa: => ZManaged[R, E, ZStream[R, E, A]])(implicit
+  private def unwrapScoped[R, E, A](fa: => ZIO[R with Scope, E, ZStream[R, E, A]])(implicit
     trace: ZTraceElement
   ): ZStream[R, E, A] =
-    managed(fa).flatten
+    scoped[R, E, ZStream[R, E, A]](fa).flatten
 
-  private def managed[R, E, A](managed: => ZManaged[R, E, A])(implicit trace: ZTraceElement): ZStream[R, E, A] =
-    new ZStream(managedOut(managed.map(Chunk.single)))
+  private def scoped[R, E, A](zio: => ZIO[R with Scope, E, A])(implicit trace: ZTraceElement): ZStream[R, E, A] =
+    new ZStream(scopedOut[R, E, Chunk[A]](zio.map(Chunk.single(_))))
 
-  private def managedOut[R, E, A](
-    m: => ZManaged[R, E, A]
+  private def scopedOut[R, E, A](
```

Comment:
I'm not sure we need these custom `scoped` and `scopedOut` implementations anymore. And I'm also not sure I translated the old implementation correctly :)